db/migrate/20221018155834_create_comments.rb

```diff
@@ -0,0 +1,9 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.text :text
+
+      t.timestamps
+    end
```

Comment:
- [ ] there should be a reference to author and post.

Reply:
Hey @mikethreels did you check the `Schema.rb` ?